go-controller/pkg/ovn/controller/apbroute/external_controller_policy_test.go

```diff
@@ -652,10 +666,9 @@ var _ = Describe("OVN External Gateway policy", func() {
 			Eventually(listNamespaceInfo(), 5).Should(HaveLen(1))
 			Eventually(func() gatewayInfoList {
 				f := getNamespaceInfo(namespaceTest.Name)
-				sort.Sort(f.staticGateways)
 				return f.staticGateways
 			}, 5).
-				Should(BeEquivalentTo(
+				Should(ConsistOf(
```

Comment:
nit: passing a variadic argument of gatewayInfo instead of a gatewayInfoList would make more explicit that it is comparing a slice of gatewayInfo rather than a single gatewayInfoList

Reply:
Meaning you could use
```
Should(ConsistOf(
        {
	        gws: sets.New(staticHopGWIP),
        },
        {
	        gws: sets.New("20.10.10.1"),
        },
        {
	        gws: sets.New("20.10.10.2"),
        },
        {
	        gws: sets.New("20.10.10.3"),
        },
        {
	        gws: sets.New("20.10.10.4"),
        },
))
```

There is two equivalent forms you can use with ConsistOf
```
Expect([]string{"Foo", "FooBar"}).Should(ConsistOf("FooBar", "Foo"))
```
or
```
Expect([]string{"Foo", "FooBar"}).Should(ConsistOf([]string{"FooBar", "Foo"}))
```

While both forms are allowed to the same effect, the first one is more typical and clear. See doc of ConsistOf

Just a nit though